src/interfaces/records/handlers/records-query.ts

```diff
@@ -76,13 +76,13 @@ export class RecordsQueryHandler implements MethodHandler {
   /**
    * Fetches the records as a non-owner, return only:
    * 1. published records; and
-   * 2. unpublished records intended for the requester (where `recipient` is the requester)
+   * 2. unpublished records intended for the query author (where `recipient` is the query author)
    */
   private async fetchRecordsAsNonOwner(tenant: string, recordsQuery: RecordsQuery)
     : Promise<RecordsWriteMessageWithOptionalEncodedData[]> {
     const publishedRecords = await this.fetchPublishedRecords(tenant, recordsQuery);
-    const unpublishedRecordsForRequester = await this.fetchUnpublishedRecordsForRequester(tenant, recordsQuery);
-    const unpublishedRecordsByRequester = await this.fetchUnpublishedRecordsByRequester(tenant, recordsQuery);
+    const unpublishedRecordsForRequester = await this.fetchUnpublishedRecordsForQueryAuthor(tenant, recordsQuery);
+    const unpublishedRecordsByRequester = await this.fetchUnpublishedRecordsByAuthor(tenant, recordsQuery);
     const records = [...publishedRecords, ...unpublishedRecordsForRequester, ...unpublishedRecordsByRequester];
```

Comment:
These variable names and a few others in this file still have `Requester` even though you changed the function names.

Reply:
Thanks for seeing, changing now!